style.css

```diff
@@ -0,0 +1,976 @@
+
+		.button__item_icon {	width: auto; }
+	}
+
+	@media (max-width: 560px) {
+		.section-desc__wrap_little {	height: 385px; }
+		
+		.text__wrap_main {	padding-left: 12px; }
+
+		.text__title {	letter-spacing: 0;	}		
+
+		.text__item_small-screen,
+		.text__item_small-bag {
+			margin-right: 10px;
+			padding-left: 4px;
+		}
+
+		.text__item_small-wifi {	margin-right: 5px;	}
+
+		.button {	width: 90%;	}
+
+		.button__item_big-right {
+			width: 30%;
+   		height: 42px;
+		}
+	}
+
+	@media (max-width: 500px) {
+		.text__item_icon {	max-width: 80%; }
+	}
+
+	@media (max-width: 445px) {
+		.section-desc__wrap_little {	height: 322px; }
+	}
+
+/* Телефон */
+	@media (max-width: 375px) {
+		.section-desc {	height: 626px; }
+		.section-desc__wrap_up {
+			background-size: cover;
+  		background-position: -35px 5px;
+		}
+
+		.section-desc__title,
+		.text__wrap_character,
+		.button {	display: none; }
+
+		.text {	height: 305px; }
+
+		.text__wrap_main {	
+			width: auto;
+			padding-left: 0;
+		}
+		
+		.text__wrap_title {
+			margin-bottom: 9px;
+  		padding-top: 17px;
+		}
+
+		.text__title {
+			text-align: center;
+  		padding-right: 7px;
+ 			letter-spacing: .6px;
+		}
+
+		.text__content {
+			width: 85%;
+	    height: 238px;
+			margin: auto;
+	    letter-spacing: .6px;
+		}
+
+		.button__item_alt-right {
+			width: 45%;
+  		height: 37px;
+			padding-right: 12px;
+		}
+	}
+
+/* Адаптив третьей секции */
+	
+/* Desktop */
+	@media (max-width: 1400px) {
+		.star {	width: 93%;	}
+
+		.star__title {	padding-top: 5px; }
+
+		.score {	width: 80%; }
+
+		.score__item_icon {	margin-right: 15px;	}
+	}
+
+	@media (max-width: 1290px) {
+		.text__title_down {
+			padding-right: 0;
+			padding-left: 30px;
+		}
+
+		.text__wrap_character-down {	width: 90%;	}
+	}
+
+	@media (max-width: 1220px) {
+		.star {	width: 97%;	}
+
+		.score {	width: 85%;	}
+
+		.section-desc__wrap-down {	background-size: contain;	}
+
+		.safe-body__item_corps {	right: 70px;	}
+	}
+
+	@media (max-width: 1150px) {
+		.text__wrap_character-down {
+			justify-content: flex-start;
+			margin-bottom: 0;
+		}
+
+		.text__wrap_subtitle {	margin-bottom: 15px;	}
+		
+		.text__wrap_section {	height: 140px;	}
+
+		.star,
+		.button_down {	width: auto;	}
+
+		.star__title {	padding-left: 0;	}
+
+		.star__item {
+			width: 23px;
+			margin-right: 1px;
+		}
+
+		.score {
+			width: 90%;
+			padding-left: 33px;
+		}
+
+		.score__item_icon {
+			width: 44px;
+			margin-right: 9px;
+  		padding-bottom: 0;
+		}
+
+		.button__item_big-down {	padding-left: 14px;	}
+
+		.section-desc__wrap-down {	background-position: 99% 23%;	}
+
+		.safe-body__item_corps {
+			top: 200px;
+  		right: 40px;
+		}
+	}
+
+	@media (max-width: 1030px) {
+		.text__wrap_character-down {	height: 205px;	}
+
+		.score {
+			margin-bottom: 30px;
+			padding-left: 25px;
+		}
+	}
+
+	@media (max-width: 900px) {
+		.text__title_down {	padding-left: 0; }
+
+		.text__wrap_character-down {	width: 97%; }
+		
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Project_page/pullRequest?prid=5536520)